pkg/plugin/prometheus/setup.go

```diff
@@ -0,0 +1,61 @@
+package prometheus
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/hellofresh/janus/pkg/errors"
+	"github.com/hellofresh/janus/pkg/plugin"
+	"github.com/hellofresh/janus/pkg/proxy"
+	"github.com/hellofresh/janus/pkg/router"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	adminRouter router.Router
+)
+
+func init() {
+	plugin.RegisterEventHook(plugin.StartupEvent, onStartup)
```

Comment:
Maybe we should only register these events if Prometheus is enabled? 
I saw that you have a configuration for this but it's only being used to register the route.

Reply:
how can i access config from here?